bee-message/src/output/feature_block/mod.rs

```diff
@@ -293,25 +308,50 @@ fn validate_dependencies(feature_blocks: &[FeatureBlock]) -> Result<(), Error> {
 
 pub(crate) fn validate_allowed_feature_blocks(
     feature_blocks: &FeatureBlocks,
-    allowed_feature_blocks_kind: &[u8],
+    allowed_feature_blocks: FeatureBlockUsages,
 ) -> Result<(), Error> {
-    if feature_blocks.len() > allowed_feature_blocks_kind.len() {
-        return Err(Error::TooManyFeatureBlocks {
```

Comment:
Why was this removed ?

Reply:
I thought we could never run into this case if feature blocks are sorted, unique, and allowed?